gk/main.c

```diff
@@ -726,18 +729,22 @@ int
 cleanup_gk(struct gk_config *gk_conf)
 {
 	unsigned int i;
-	unsigned int num_lcores = gk_conf->lcore_end_id - gk_conf->lcore_start_id + 1;
+	unsigned int num_lcores = gk_conf->lcore_end_id -
+		gk_conf->lcore_start_id + 1;
+
 	/*
 	 * Atomically decrements the atomic counter (v) by one and returns true 
 	 * if the result is 0, or false in all other cases.
 	 */
 	if (rte_atomic32_dec_and_test(&gk_conf->ref_cnt)) {
 		for (i = 0; i < num_lcores; i++) {
 			if (gk_conf->instances[i].ip_flow_hash_table)
-				rte_hash_free(gk_conf->instances[i].ip_flow_hash_table);
+				rte_hash_free(gk_conf->instances[i].
```

Comment:
80 is OK (according to the style guidelines), but I think less than 80 is preferable.